xml/ses_glossary.xml

```diff
@@ -25,22 +25,85 @@
     </para>
    </glossdef>
   </glossentry>
+  <glossentry><glossterm>&alertmanager;</glossterm>
+   <glossdef>
+    <para>
+     A single binary which handles alerts sent by the &prometheus; server and
+     notifies the end user.
+    </para>
+   </glossdef>
+  </glossentry>
+  <glossentry><glossterm>archive sync module</glossterm>
+   <glossdef>
+    <para>
+     Module that enables creating an &ogw; zone for keeping the history of S3
+     object versions.
+    </para>
+   </glossdef>
+  </glossentry>
   <glossentry><glossterm>Bucket</glossterm>
    <glossdef>
     <para>
      A point that aggregates other nodes into a hierarchy of physical
      locations.
     </para>
     <important>
-     <title>Do Not Mix with S3 Buckets</title>
      <para>
-      S3 <emphasis>buckets</emphasis> or <emphasis>containers</emphasis>
-      represent different terms meaning <emphasis>folders</emphasis> for
-      storing objects.
+      Not to be confused with S3 <emphasis>buckets</emphasis> or
+      <emphasis>containers</emphasis>.
```

Comment:
OK, reword appreciated, but TBH I still do not feel it clearly disambiguates the term(s).

So, for example, based on 15min on Google, reading various bits of docs including ours, Amazon's, OpenStack's and others, I very tentatively suggest something like:

«
Be careful not to confuse the SES meanings of "buckets" and "containers" with uses of these terms by other products. 

OGW buckets are lists of OSDs, used to collect devices into physical locations. This is entirely distinct from the term as used in the context of Amazon S3.

S3 buckets are closer to what are termed containers in the context of OpenStack Swift. Here, containers are the entities that store objects, and must not be confused with operating system containers, for example those managed by Docker or CRIO.
»

This is based on my very fragmentary understanding.

What I am trying to get at is that to successfully make the distinction clear between "word X in context 1" versus "word X in context 2", we need to spell out _both_ contexts and explain the difference.

Worse still is what we seem to have here: where we _also_ have word Y in context 3 and word Y in context 4, and Word X.1 is the same **word** as Word X.2 but actually what it _means_ is the same as word Y.2.

If I have this right: 

An OpenStack Container is similar to an S3 Bucket. This is totally different from an OGW Bucket or a Docket Container.

Please don't take my word for it, though. All I am saying is that if we have buckets (2 different meanings) and containers (2 different meanings), we need to explicitly say:

"Bucket" in our context means "container" in their context. That does not mean the same as "container" in THAT context.

I submit that these days, any competent Linux techie knows about Docker/Rancher/K8s-type containers. They may not know OpenStack or S3. Personally I have used Docker but never touched OpenStack or S3.

Reply:
It is equally safe to say that any engineer who is familiar with docker/k8s, are familiar with OpenStack. That's not the question here. I will muse on this further, but when it comes down to it: Those looking for glossary-type clarifications in our SES docmentation should be already familiar with what this is.

I'm actually going to remove this, this hinders rather than helps. This should be a definition list. Explanations and contextual understanding goes in the product docs.